gradle.properties

```diff
@@ -14,6 +14,9 @@ POM_LICENCE_DIST=repo
 POM_DEVELOPER_ID=mapbox
 POM_DEVELOPER_NAME=Mapbox
 
+POM_UI_SNAPSHOT_VERSION_NAME=1.0.0-SNAPSHOT
+POM_CORE_SNAPSHOT_VERSION_NAME=1.0.0-SNAPSHOT
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/pull/2623#discussion_r396428277 addressed by ☝️

Reply:
It'd be great to add `publish-local` entries to `Makefile` but definitely not blocking this PR, feel free to include here or in a different one 👍

Reply:
Merging this PR. Find an issue where I am not able to run `dokka` tasks for core. Let us address this in a separate PR along with `Makefile` changes.